### ISSUE TITLE
Cannot connect to Power BI dataset

### ISSUE THREAD

**Issue by bdarbonneau** [microsoft/Analysis-Services#48]
I am using v5.0.5 on W10. I have 
- enabled XML read/write on the Premium P1 capacity
- enabled in pbd the enhanced metatdata option
- published a dataset to a workspace in premium capacitiy
- copied the xmla url from the service
When I attempt to connect, I can authenticate, I can select in the dropdown list box the target dataset, but then I get this error: an internal error has occured
Any idea what I could do to solve this ? 
![Image 386](https://user-images.githubusercontent.com/32508734/78382101-93b84680-75d6-11ea-95db-007e79b885ea.png)

**Comment by christianwade** [comment]
Hi, I believe this is the same issue as below. This will require a new version of the AS client libraries which will take a few weeks. In the meantime, the workaround (removal of query groups) is discussed in the item below. Since so many people are hitting this, will likely put a more informative error message ...

https://github.com/microsoft/Analysis-Services/issues/47

**Comment by bdarbonneau** [comment]
I upgraded from v5.0.2 to v5.0.3 at it seems to have solved the issue. I can now use this great tool. Thks very much.
However, after pasting the dataset url and going through the authentication process, I still get an error 'cannot connect to the database'. But this little trick does the job: I click the Compare button, and explicitely re-select the dataset in the dataset dropdown listbox. It's like the 'initial catalog' parameter of the connection string that I had pasted into the workspace text box had not been picked up correctly.
